### DISCUSSION TITLE
I surprised that I can't resize the image immediately

### DISCUSSION THREAD

**Discussion by Dann1y** [GitbookIO/community#181] [Feature Requests]
### Ask a question, start a conversation, or make an annoucement

https://docs.gitbook.com/content-creation/blocks/insert-images

I read this document and tried to resize the image in the gitbook editor, but it didn't work at all and I don't know how.

And I was surprised again that there is no way to resize images by drag and drop, which is supported by most services.

I usually insert image by copy and paste but I think If i did like that, the image inserted like a block so I couldn't resize.

How do you resize and use images?
The experience with images wasted a lot of my time.

**Comment by aikoii** [comment]
Hi @Dann1y,

Great news! 😀 

We've just launched a new feature that allows you to resize images with ease. With Git Sync, you can specify the percentage or pixel value for the resize as well.

We've also made sure to update our documentation accordingly.

Thanks for your feedback! 👋

**Reply by Dann1y** [reply]
<img width="304" alt="image" src="https://user-images.githubusercontent.com/29726020/236663227-2959d3ba-e11e-45a6-b2d3-f9be1a7e4534.png">
Now, I can resize the image what I copied and pasted.
Thank you so much to the development team for adding this feature.😊
